staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/lazy_fieldmanager_test.go

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package fieldmanager_test
+
+import (
+	"testing"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager"
+)
+
+type fakeObjectCreater struct{}
+
+var _ runtime.ObjectCreater = &fakeObjectCreater{}
+
+func (_ *fakeObjectCreater) New(_ schema.GroupVersionKind) (runtime.Object, error) {
+	return &unstructured.Unstructured{Object: map[string]interface{}{}}, nil
+}
+
+func TestUpdateOnlyDoesNotTrackManagedFields(t *testing.T) {
```

Comment:
What about testing that apply starts tracking?
Conflicts with past updated fields?